.github/workflows/dockerpublish.yml

```
@@ -0,0 +1,99 @@
+name: Docker
+
+on:
+
+  # Publish releases as 'latest'.
+  release:
+    types: [published]
+
+  push:
+    # Publish `master` as Docker `master-latest` image
+    # Publish for any branch as well
+    branches:
+      - master
+      - '*'
+
+    # Publish `v1.2.3` tags as releases.
+    tags:
+      - 'v*'
+
+  # Run tests for any PRs.
+  pull_request:
+
+
+env:
+  IMAGE: cluster.dev
+  REGISTRY: docker.io/clusterdev
+
+jobs:
+  # Run tests
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Run tests
+        run: |
+          docker build . 
```

This can be simplified
```suggestion
        run: docker build . 
```